subscriber.rb

```diff
@@ -0,0 +1,47 @@
+require 'aws-sdk-sqs'
+require_relative 'load_aws'
+
+class Subscriber
+
+  attr_reader :sqs
+
+  def initialize
+    creds = Aws::Credentials.new(AWS["access_key_id"], AWS["secret_access_key"])
+    @sqs = Aws::SQS::Client.new(region: AWS["region"], credentials: creds)
+  end
+
+  def valid_sleep_period?(sleep_period)
+    sleep_period = '2'
```

Comment:
As mentioned earlier - set the `sleep_period` in the method definition. Set it to `0`.
So  `def valid_sleep_period?(sleep_period = 0)`

Reply:
the problem with `def valid_sleep_period?(sleep_period = 0)` is that if the user does not enter the sleep period, we get `nil` when we access as `options[1]` for an array of size 1. In that case, it overwrites the sleep_period value as nil and we lose the '0'.

Reply:
I think the ternary with nil check is appropriate here.

Reply:
Ok.. good point.. this is affected by L23